### DISCUSSION TITLE
Setup custom domain from ovh

### DISCUSSION THREAD

**Discussion by AdiXsus** [community/community#32035] [Other Feature Feedback, Questions, & Ideas] [answered]
can I set a page on the adixsus.xyz domain which is hosted on ovh.com?
If so, please ask for help in setting it up.

**Comment by BLKKKBVSIK** [comment, marked as answer]
Yes, you can totally do it.

Two ways:
Either you make a CNAME record on OVH that redirect to your GitHub pages.
Either you add a domain to your GitHub Pages. You can find more informations [here](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site)
